README.md

```diff
@@ -93,7 +93,8 @@ showcaseManager.show(context) // or showcaseManager.show(context, REQUEST_CODE_S
 | `builder.setSlidableContentList(List<SlidableContent>)`            | show slidable content                | yes      | null                         | no       |  
 | `builder.build()`                              | will return ShowcaseManager instance                     | no       |                              |          |
 | `showcaseManager.show(Context)`                | show the tooltip with set attributes on                  | no       |                              |          |
-
+| `builder.showDurationMillis(Long)`             | duration of the tooltip visibility                       | yes       | 2000L                       | no       |
```

Comment:
You can move this lines above `builder.build()` line.

Reply:
Done [here](https://github.com/Trendyol/showcase/pull/26/commits/08054099a6c2d11f0ba3ae4f984fd120b0b78b86)